package-lock.json

```diff
@@ -3,6 +3,5707 @@
   "version": "5.10.0",
   "lockfileVersion": 1,
```

Comment:
It seems in b2f1beb968471771d4ec40633fc79eea7edd0648 you just changed this line, could you please revert all changes in this file? If you're having trouble with this, let me know so that I can fix it for you before this is merged :slightly_smiling_face:

Reply:
Hi @ericcornelissen, I revert all the changes in package-lock.json file. But this time all the checks have been failed. Can you please point out if I am doing some mistake over here??

Reply:
Seems neither 5d12461ee0b99307621427a7f87d1f8e506fcc59 nor b60301af938fefed08ca368955ece5eceba03c0d did the trick :sweat_smile: Actually, with #6179 being merged you should now be able to just remove the `package-lock.json` file and not have to worry about it at all! (still, if you need help, feel free to ask :slightly_smiling_face:)